db/seeds.d/150-module_job_templates.rb

```diff
@@ -1,7 +1,7 @@
 if Katello.with_remote_execution?
   User.as_anonymous_admin do
     JobTemplate.without_auditing do
-      module_template = JobTemplate.find_by(name: 'Module Action - SSH Default')
+      module_template = JobTemplate.find_by(name: 'Module Action - Script Default')
```

Comment:
Do we need a migration to change the name as well for existing installs?

Reply:
It is in rex